### ISSUE TITLE
Add option to return only filename of asset, without preceding path

### ISSUE THREAD

**Issue by isellsoap** [sueddeutsche/stats-replace-webpack-plugin#2]
It would be nice if there was an option in a replacer object to say something like "basename: true" or something. This would have the effect that only the file name of the chunk would be returned without the preceding path. This would greatly increase flexibility.

It would require to add [path](https://www.npmjs.com/package/path) as a dependency and change https://github.com/sueddeutsche/stats-replace-webpack-plugin/blob/master/lib/index.js#L63 to something like

```
 return replacer.basename ? path.basename(relatedAsset) : relatedAsset;
```

`replacer.basename` reflects the exact name of the appropriate method of the [path package](https://www.npmjs.com/package/path), this can of course also be another name.

What do you think about that? Do you think this would be useful?

**Comment by benurb** [comment]
I'm a bit unsure about this. Currently the plugin afaik includes the path relative to the configured publicPath. Why would you want to use the basename when the linked css/js file does not exist in this location?
Anyway, I'll implement this 😄 👍

**Comment by isellsoap** [comment]
I don’t know if my setup is so complicated, but I do put the compiled assets in a complete other location than the source files and somehow I couldn’t manage to do it. Having the option to only output the basename would be a perfect solution for me. :-)

**Comment by isellsoap** [comment]
Also, there could be situations where you want to reference an absolute URL of the asset in the view and thus only want the file name of the asset and not any potentially relative paths.

**Comment by benurb** [comment]
Sorry, this took way too long 👎 
But finally I found time to add it. Thanks again for your suggestion 🙂 
Published to npm as version 1.2.0.
